app/classes/Transvision/ShowResults.php

```diff
@@ -397,6 +412,10 @@ public static function resultsTable($search_object, $search_results, $page)
             $anchor_name = str_replace(['/', ':'], '_', $key);
 
             $clipboard_target_string  = 'clip_' . md5($target_string);
+            if ($locale2 == 'sr') {
+                $transliterated_string = self::getTransliteratedString($target_string, 'sr-Cyrl');
+                $transliterate_id = substr($clipboard_target_string, 5);
```

Comment:
The issue i have is that I have to identify the Cyrillic string and the transliterated one with nearly the same id because the button must be associated with its corresponding texts and I use the id of the button to find the right text to toggle [here](https://github.com/SkySymbol/transvision/blob/button_transliterate/web/js/toggle_transliterated_string.js#L3-4). And the ```<div>``` for the target string is already identified with ```$clipboard_id```.

I don't know if it's the right approach.